### PR TITLE
[bitnami/grafana-operator] Add additional labels to pods in the grafana deployment

### PR DESCRIPTION
**Pull request by kopendrex**
### Description of the change

Added missing labels in the Grafana pod templates. Right now there is no possibility to add labels in this section other than by using common labels. This solution is too wide.

### Benefits

Now it's possible to set pod labels for pods in the grafana deployment by spec template.

### Possible drawbacks

Maybe `podLabels` could be named differently.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the PR! Could you fix the DCO issue?

**Comment by kopendrex** [comment]
Hi, @javsalgar any update?

File: bitnami/grafana-operator/Chart.yaml [hunk 1]
```diff
@@ -30,4 +30,4 @@ maintainers:
 name: grafana-operator
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/grafana-operator
-version: 4.1.0
+version: 4.1.1
```
Comment (jotamartos) [inline bitnami/grafana-operator/Chart.yaml:33]: As you are adding a new parameter, we need to bump the version to a new minor one.

```suggestion
version: 4.2.0
```

**Review by jotamartos — COMMENTED** [review]
Adding suggestions

**Review by jotamartos — DISMISSED** [review]

**Review by jotamartos — APPROVED** [review]

**Pull request merged**